texas_holdem_poker.py

```diff
@@ -7,6 +7,13 @@
 import queue
 import time
 
+VALUES = [
+    "TWO", "Three", "Four", "Five", "Six", "Seven", "Eight", "Nine", "Ten",
+    "Jack", "Queen", "King", "Ace"
+]
+VALUE_NAME_MAP = dict(zip(range(12), VALUES))
+SUIT_MAP = dict(zip(range(4), ["Diamonds", "Clubs", "Hearts", "Spades"]))
```

Comment:
I think dictionary comprehension would be better here. It's more readable and gets rid of the need for zip()
This would reqauire you to have a similar variable to VALUES, which would hold the suits.
`SUITS = ["Diamonds", "Clubs", "Hearts", "Spades"]`
Also, we won't ever need to change the values in these 2 data structures (lists). So tuples would be better, no?

Reply:
`dict(zip(` is a canonical way of building a dict from two iterables: https://stackoverflow.com/a/209854. 

But, I totally agree that tuples would be better.

Reply:
tuples added.

Reply:
You're right, I figured dict comprehension would be faster to perform, but I tested it and I was wrong, apologies!

Reply:
Dict comprehension is not faster? @Sean

> You're right, I figured dict comprehension would be faster to perform, but I tested it and I was wrong, apologies!

Dict comprehension is not faster than zip?

Reply:
From the test I've done, it appears so!

Reply:
When I tested it, it wasn't no, but now I'm doubting if I tested it right the more I think about it..

Also just had a thought, is `VALUE_NAME_MAP = {k:v for k,v in enumerate(VALUES)}` not more pythonic?